src/torrent.cpp

```diff
@@ -124,9 +124,9 @@ namespace libtorrent
 		return ret;
 	}
 
-	template <typename T, typename IndexType = int>
-	aux::vector<T, IndexType> clone(aux::vector<T, IndexType> const& v)
-	{ return v; }
+	template <typename T, typename IndexType>
+	aux::reverse_const_wrapper<T> safe(aux::vector<T, IndexType> const& v)
```

Comment:
I wanted to differentiate, this "`safe`" from `aux::reverse`, but I know it's a little odd the naming